templates/publisher/register-name.html

```diff
@@ -41,27 +46,26 @@ <h1>Register a snap name</h1>
     <div class="row">
       <div class="col-8">
         <div class="p-form-validation">
-          <label for="snap-title">Is Private:</label>
+          <label for="public">Snap privacy</label>
+          <p class="p-form-help-text">This can be changed at any time after the initial upload</p>
           <div class="p-form-validation__field">
-            <input name="is_private" class="p-form-validation__input" type="checkbox">
+            <input type="radio" name="is_private" value="public" class="p-form-validation__input" id="public" {% if not is_private %} checked {% endif %}>
+            <label for="public">Public</label>
+            <input type="radio" name="is_private" value="private" class="p-form-validation__input" id="private" {% if is_private %} checked {% endif %}>
+            <label for="private">Private</label>
           </div>
         </div>
       </div>
     </div>
 
     <div class="row">
-      <div class="col-8">
-        <div class="p-form-validation">
-          <label for="snap-title">Registrant Comment:</label>
-          <div class="p-form-validation__field">
-            <textarea class="p-form-validation__input u-no-margin" name="registrant_comment" rows="10"></textarea>
-          </div>
-        </div>
-      </div>
+      <hr/>
     </div>
 
     <div class="row">
-      <div class="col-4">
+      <div class="u-align--right">
+
+        <a class="p-button--neutral" href="/account/snaps">Cancel</a>
         <input type="submit" class="p-button--positive" value="Register"/>
```

Comment:
Second button has unnecessary margin top (because Vanilla). Adding `u-no-margin--top` to it fixes it.